### ISSUE TITLE
Timeouts while an archive is mounted

### ISSUE THREAD

**Issue by stratosgear** [KenKundert/emborg#48]
I found out that while you have an archive mounted you get errors while trying to create new backups:

```
➜ emborg -c linked_path create
emborg error: borg create:
    Failed to create/acquire the lock /home/stratos/borg_backups2/serenity-stratos-linked_path/lock (timeout).
```

The message was confusing enough and it took me QUITE some time until I figured out that it might be due to the currently mounted archive.  Once I emborg unmounted the archive the timeouts were gone.

Can this particular case be captured, and maybe a more informative message shown?  Maybe even something "Do you currently have any archives mounted?".

**Comment by KenKundert** [comment]
That message is produced by *Borg* and simply passed to the user by *Emborg*.  Anything I could do would be relatively fragile.  The best I can do is look for that error message and add a comment if it is returned by *Borg*.  If the error message ever changes, this enhancement could fail.

**Comment by KenKundert** [comment]
I think this is a good request, and have improved the error message as you suggested.  Concerning the fragility, I have added a test that will catch the missing message if Borg decides to change their message. The updated version is available on GitHub.

**Comment by KenKundert** [comment]
The release that fixes this issue (1.25) has been pushed to pypi.
